restcomm/restcomm.application/src/main/webapp/WEB-INF/scripts/mariadb/sql/accounts.xml

```diff
@@ -28,6 +28,10 @@
 		SELECT * FROM restcomm_accounts WHERE parent_sid=#{parent_sid};
```

Comment:
what is this mariadb script? is this something that leaked during testing?

im trying to switch all files generated during test into maven "target" folder, so its inmediately ignore by git, and we dont get into this situations. can we change where the db is started for tests?

Reply:
@jaimecasero both mariadb/mysql and hsql have slightly different syntax that is why we have 2 seperate query scripts.
in testsuit we use hsql scripts only